src/Nethermind/Nethermind.Crypto/Nethermind.Crypto.csproj

```diff
@@ -47,15 +47,6 @@
 <!--        <Content Include="runtimes\win-x64\native\eth_pairings2537.dll">-->
 <!--            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>-->
 <!--        </Content>-->
-      <Reference Include="Microsoft.AspNetCore.DataProtection.Abstractions, Version=3.1.5.0, Culture=neutral, PublicKeyToken=adb9793829ddae60">
-        <HintPath>..\..\..\..\..\Program Files\dotnet\packs\Microsoft.AspNetCore.App.Ref\3.1.5\ref\netcoreapp3.1\Microsoft.AspNetCore.DataProtection.Abstractions.dll</HintPath>
-      </Reference>
-      <Reference Include="Microsoft.AspNetCore.DataProtection.Extensions, Version=3.1.5.0, Culture=neutral, PublicKeyToken=adb9793829ddae60">
-        <HintPath>..\..\..\..\..\Program Files\dotnet\packs\Microsoft.AspNetCore.App.Ref\3.1.5\ref\netcoreapp3.1\Microsoft.AspNetCore.DataProtection.Extensions.dll</HintPath>
-      </Reference>
-      <Reference Include="System.Security.Cryptography.ProtectedData, Version=4.0.5.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-        <HintPath>..\Nethermind.Runner.Test\bin\Debug\netcoreapp3.1\System.Security.Cryptography.ProtectedData.dll</HintPath>
-      </Reference>
```

Comment:
Not updated to .NET 5, and everything still *appears* to work when they are missing.